### PR TITLE
Add onAliasQueried

### PR DESCRIPTION
**Pull request by Half-Shot**
Add onAliasQueried to retrieve a room ID after creation via onAliasQuery.

### CODE REVIEW

**Comment by Kegsay** [comment]
ok to test

File: lib/bridge.js [hunk 1]
```diff
@@ -35,6 +35,9 @@ var util = require("util");
  * the bridge will invoke this function when queried via onAliasQuery. If
  * not supplied, no rooms will be provisioned on alias queries. Provisioned rooms
  * will automatically be stored in the associated <code>roomStore</code>.
+ * @param {Bridge~onAliasQueried=} opts.controller.onAliasQueried Function. If supplied,
+ * the bridge will invoke this function when queried via onAliasQueried. Will
+ * be called when a room has been created via onAliasQuery.
```
Comment (Kegsay) [inline lib/bridge.js]: I think you mean:

> If supplied, the bridge will invoke this function when a room has been created via onAliasQuery.

**Comment by Kegsay** [comment]
ok to test

**Comment by Kegsay** [comment]
ok to test

**Comment by Half-Shot** [comment]
I am forever at odds with these... *frantically fixes*

**Comment by Kegsay** [comment]
LGTM!

**Pull request merged**